spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/service/impl/TaskConfigurationProperties.java

```diff
@@ -41,9 +41,26 @@
 	@NotBlank
 	private String composedTaskRunnerName = "composed-task-runner";
 
+	/**
+	 * The schedule application name to be used for the scheduler task launcher.
+	 */
 	@NotBlank
 	private String schedulerTaskLauncherName = "scheduler-task-launcher";
 
+	/**
+	 * The URI of the resource to be used for launching apps via a schedule.
+	 */
+	private String schedulerTaskLauncherUrl = "maven://org.springframework.cloud:spring-cloud-dataflow-scheduler-task-launcher:2.3.0.BUILD-SNAPSHOT";
```

Comment:
We could leave this empty and set value in `dataflow-server-defaults.yml` where we can replace value via maven build with `@project.version@`. Though this has an annoying issue that when running via IDE that value would be wrong.